scripts/legislation/africa/egypt_english.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^FINAL PROVISIONS (ARTICLES 102 to 104).*$/,$p' | \
+    sed -n '/^PRELIMINARY PART/,$p'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
```

Comment:
Do you want the caret (`^`) in the regular expression?

Reply:
No, I think this is an inefficient expression as previously discussed. But I would prefer to leave it as changing it now may result in changes across the rest of the code and text.